napari/_qt/widgets/qt_keyboard_settings.py

```diff
@@ -231,6 +231,7 @@ def _set_table(self, layer_str: str = ''):
                 # Set action description.  Make sure its not selectable/editable.
                 item = QTableWidgetItem(action.description)
                 item.setFlags(Qt.ItemFlag.NoItemFlags)
+                item.setFlags(item.flags() | Qt.ItemIsEnabled)
```

Comment:
why not 
```suggestion
                item.setFlags(Qt.ItemFlag.ItemIsEnabled)
```

as `NoItemFlags` is equal to zero: https://doc.qt.io/qt-6/qt.html#ItemFlag-enum

Reply:
was copy-pasta 😬 
Your way is better 😊

Reply:
wait, deleting `item.setFlags(Qt.ItemFlag.NoItemFlags)` could change things...